bundle/edu.gemini.pit.launcher/src/main/scala/edu/gemini/pit/launcher/PITLauncher.scala

```diff
@@ -17,8 +17,11 @@ object PITLauncher extends App {
   val locale = Locale.getDefault
   Locale.setDefault(Locale.ENGLISH)
 
-  // We normally want to be in test mode in development
-  System.setProperty("edu.gemini.pit.test", "true")
+  // Check first on the env or else default to true
+  Option(System.getProperty("edu.gemini.pit.test")) match {
+    case None => System.setProperty("edu.gemini.pit.test", "true")
+    case _ => ()
+  }
```

Comment:
In `AppMode.scala` we're treating undefined as false, so these bits of code contradict each other. The default app distribution does not set the property at all, so with this change it will default to true.

Reply:
Yeah, this was acting wonky, and we were purposefully removing it, IIRC, to represent `false`.

Reply:
I'm not sure I follow

Reply:
You're right about `AppMode`, OTOH `AppMode` is only used from the Activator, maybe I should remove it altogether

Reply:
`AppMode` is used also in the `ShellAdvisor`, so yes these changes need to be synchronized

Reply:
Confusion. Since it defaults to `false` anyway does this change actually accomplish anything?